### ISSUE TITLE
Wrong integrity hash code in management/tub_web/base.html

### ISSUE THREAD

**Issue by coolerking** [wroscoe/donkey#313]
I tried tubclean command with donkey 2.5.1.
execute 'donkey tubclean data' and open http://localhost:8886/

There are 4 error messages on web console like:

> Failed to find a valid digest in the 'integrity' attribute for resource 'http://localhost:8886/static/jquery-3.1.1.min.js' with computed SHA-256 integrity 'HPMOWdIdSuVgr3FD9ZE+/MgiK8qk/MdQjrgCtfqp6U4='. The resource has been blocked.

please omit integrity or re-generate hash in management/tub_web/base.html

p.s.
Maybe, there is a typo in setup.py, line16:
  download_url='https://github.com/wroscoe/donkey/archive/2.1.5.tar.gz',

**Comment by Schmill** [comment]
I don't believe that this was regarding the web controller, but rather the web interface used for tub_clean, (which also seems to have been removed from the codebase - see my ticket #394 for further info ; I can do a pull request for re-adding the tubclean files and correcting the hashes.
